oceania/README.OCEANIA

```diff
@@ -86,3 +86,134 @@ of the grid file will give results agreeing to within 1cm 99.97% of the time.
+
+These grids represent the systematic errors in the local vertical datums. 
+The local vertical datums are each based on levelling from a tide gauge.  The include systematic errors from the levelling as well as a potential of the local tidal signal at the port from the global mean gravitational equipotential surface.  The NZVD2016 datum is based on gravity measurements from a national airborne gravity program and is much less prone to 
+regional systematic errors.  The transformation grids are derived from comparing the benchmark heights at which both levelling data and GNSS (Global Navigation Satellite system) data are available.  
+
+The height determined from the grid model is added to an NZVD2016 height to obtain
+the corresponding local vertical datum shift.
+
+The following grids are available:
+
+*Local vertical datum*: Auckland 1946
+*Filename*: auckht1946-nzvd2016.gtx
+
+*Local vertical datum*: Bluff 1955
+*Filename*: blufht1955-nzvd2016.gtx
+
+*Local vertical datum*: Dunedin 1958
+*Filename*: duneht1958-nzvd2016.gtx
+
+*Local vertical datum*: Dunedin-Bluff 1960
+*Filename*: dublht1960-nzvd2016.gtx
+
+*Local vertical datum*: Gisborne 1926
+*Filename*: gisbht1926-nzvd2016.gtx
+
+*Local vertical datum*: Lyttelton 1937
+*Filename*: lyttht1937-nzvd2016.gtx
+
+*Local vertical datum*: Moturiki 1953
+*Filename*: motuht1953-nzvd2016.gtx
+
+*Local vertical datum*: Napier 1962
+*Filename*: napiht1962-nzvd2016.gtx
+
+*Local vertical datum*: Nelson 1955
+*Filename*: nelsht1955-nzvd2016.gtx
+
+*Local vertical datum*: One Tree Point 1964
+*Filename*: ontpht1964-nzvd2016.gtx
+
+*Local vertical datum*: Stewart Island 1977
+*Filename*: stisht1977-nzvd2016.gtx
+
+*Local vertical datum*: Taranaki 1970
+*Filename*: taraht1970-nzvd2016.gtx
+
+*Local vertical datum*: Wellington 1953
+*Filename*: wellht1953-nzvd2016.gtx
+
+*Reference*: https://www.linz.govt.nz/data/geodetic-system/datums-projections-and-heights/vertical-datums/vertical-datum-relationship-grids
+
+#### Usage
+
+PROJ versions prior to 6.0 do not support vertical grid shifts.  However this may be 
+emulated using the +geoidgrids, which subtracts the height offset defined in the grid
+from height (rather than adding it for a vertical grid shift offset).  So the conversions
+can be treated as geoid grids converting from the local vertical datum to NZGD2000.
+
+To convert from a local vertical datum height to a NZVD2016 height.
+
+   echo '175 -37 0' | cs2cs -v -f %.8f +proj=latlong +ellps=GRS80 +to +proj=latlong +ellps=GRS80 +geoidgrids=auckht1946-nzvd2016.gtx
+
+To convert from a NZVD2016 height to a local vertical datum height.
+
+   echo '175 -37 0' | cs2cs -v -f %.8f +proj=latlong +ellps=GRS80 +geoidgrids=auckht1946-nzvd2016.gtx +to +proj=latlong +ellps=GRS80 
+
+[PROJ]: https://proj.org
+[Land Information New Zealand]: https://www.linz.govt.nz
+[Creative Commons Attribution 4.0 International licence]: https://data.linz.govt.nz/license/attribution-4-0-international/
```

Comment:
this is just a definition, but there's no link to that license. Look at the Australian entries for examples of a License: field

Reply:
Thanks - Will fix that too ...  I thought it was there but must have lost it in the file merge :-(